clusters/app.ci/supplemental-ci-images/cincinnati-ci/admin_rbac.yaml

```diff
@@ -80,18 +68,6 @@ items:
     subjects:
       - kind: ServiceAccount
         name: ci-image-sa
-  - kind: RoleBinding
-    apiVersion: rbac.authorization.k8s.io/v1
-    metadata:
-      name: cincinnati-ci-public-admin-binding-cluster
-      namespace: cincinnati-ci-public
-    roleRef:
-      apiGroup: rbac.authorization.k8s.io
-      kind: ClusterRole
-      name: admin
```

Comment:
Shouldn't this be the `pull-secret-namespace-manager`?

Reply:
Updated.